api/src/main/java/org/opencord/aaa/AaaConfig.java

```diff
@@ -85,7 +85,19 @@
     // Packet Customizer Default value
     protected static final String DEFAULT_PACKET_CUSTOMIZER = "default";
 
-
+    //Event publisher schedule value initial delay
+    private static final int INITIAL_DELAY = 0;
+    private static final int REPEAT_DELAY = 20;
```

Comment:
Instead of REPEAT_DELAY defined as a constant here; better make it as a configurable property in AaaStatisticsManager, where it is to be used. Also, use a more appropriate name like StatisticsGenerationInterval or StatisticsWriteoutFrequency. Refer to configuration mechanism available for this on wiki page  https://wiki.onosproject.org/display/ONOS/Component+Configuration